test/Gather_Metrics.js

```diff
@@ -206,7 +206,7 @@ contract("Gather Metrics", (accounts) => {
       console.log("Gas Results:")
 
       let signer_list = [accounts[0]]
-      for(let signer_count = 1; signer_count < accounts.length; signer_count++){
```

Comment:
I see we create 50 accounts now, I guess we can keep this as is, 50 validators is our approx expectation at first when we open the validators set, so would be good to see the gas cost for up to this number.

Reply:
Ah yeah, I just added 50 for good measure, but then this test expected the number of accounts to be equal to the 20 private keys it generates at the top

Reply:
indeed, but if we can make it up to 50 without too much trouble, then that would be very valuable info.

Reply:
Let me do that

Reply:
cheers